docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,220 @@
+# Percona Monitoring and Management 2.43.0
+
+| **Release date** | September 16th, 2024                                                                                     |
+| -----------------| ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/index.md) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## What's new in this release
+
+PMM 2.43 brings a host of updates, including redesigned dashboards, new collectors and metrics, enhanced troubleshooting capabilities, strengthened security, and expanded platform support with experimental ARM compatibility for PMM Client.
```

Comment:
"expanded platform support" sounds like it's "already done" but we just starting. maybe some softer statement like "try, test we started road to  ARM "